crates/aptos-api-tester/src/tests/new_account.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    consts::FUND_AMOUNT,
+    fail_message::{
+        ERROR_COULD_NOT_CREATE_ACCOUNT, ERROR_COULD_NOT_FUND_ACCOUNT, ERROR_NO_ACCOUNT_DATA,
+        FAIL_WRONG_ACCOUNT_DATA,
+    },
+    persistent_check, time_fn,
+    utils::{
+        check_balance, create_account, emit_step_metrics, get_client, get_faucet_client,
+        NetworkName, TestFailure, TestName,
+    },
+};
+use aptos_api_types::U64;
+use aptos_logger::info;
+use aptos_rest_client::{Account, Client, FaucetClient};
+use aptos_sdk::types::LocalAccount;
+use aptos_types::account_address::AccountAddress;
+
+/// Tests new account creation. Checks that:
+///   - account data exists
+///   - account balance reflects funded amount
+pub async fn test(network_name: NetworkName, run_id: &str) -> Result<(), TestFailure> {
+    // setup
+    let (client, faucet_client, account) = emit_step_metrics(
+        time_fn!(setup, network_name),
+        TestName::NewAccount,
+        "setup",
+        network_name,
+        run_id,
+    )?;
+
+    // check account data persistently
+    emit_step_metrics(
+        time_fn!(
+            persistent_check::account,
+            "check_account_data",
+            check_account_data,
+            &client,
+            &account
+        ),
+        TestName::NewAccount,
+        "check_account_data",
```

Comment:
Nit: could probably make a const since you duplicate it to prevent different spelling mistakes